api/auth/user_management.py

```
@@ -69,125 +69,50 @@ async def delete_user_token(api_token: str):
+    except Exception as e:  # pylint: disable=broad-exception-caught
         logging.error("Error deleting user token: %s", e)
 
 
-async def ensure_user_in_organizations(provider_user_id: str, email: str, name: str, provider: str, api_token: str, picture: str = None):
+async def ensure_user_in_organizations(  # pylint: disable=too-many-arguments
+    provider_user_id: str,
+    email: str,
+    name: str,
+    provider: str,
+    api_token: str,
+    picture: str = None
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)